ES/ESmy020.xml

```diff
@@ -688,6 +690,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          <change who="MK" when="2014-10-21">last edited</change>
          <change who="PL" when="2016-05-10">transformed from mycore to TEI P5</change>
          <change who="DN" when="2020-08-20">continued description</change>
+         <change who="DN" when="2020-08-27">incorporated comments from Eugenia Sokolinski</change>
```

Comment:
one more error I missed last time:
ms_i2 title needs ref not corresp
        title type="complete" **ref**="LIT3949MashafaKrL"

Reply:
ok